src/index.js

```diff
@@ -80,13 +80,13 @@ class Circulation extends Component {
             route: 'fine-policies',
             label: <FormattedMessage id="ui-circulation.settings.finePolicy.paneTitle" />,
             component: FinePolicySettings,
-            perm: 'ui-circulation.settings.overdue-fines-policies',
+            perm: 'ui-circulation.settings.read-overdue-fines-policies',
```

Comment:
Could you please clarify this is correct that we create ui-circulation.settings.**view**-overdue-fines-policies but we check ui-circulation.settings.**read**-overdue-fines-policies
https://github.com/folio-org/ui-circulation/pull/790/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R221
The same below

Reply:
Nope, that's my error! Good catch, thank you. Fixed. (I thought of these as "read" policies and drafted them that way, but the ticket refers to them as "view" policies, so I changed the language but missed a spot.)